src/styles/index.scss

```diff
@@ -1,3 +1,111 @@
 body {
   margin: 0;
 }
+
+.card {
+  max-width: 200px;
+  height: 406px;
```

Comment:
Don't apply fixed sizes on block. Width is okay bcs its explicitly said that card have 200 px. Let the content size dictate it. To avoid overflow or accidental scroll bar